kubernetes/clickhouse/templates/clickhouseKeeper.yaml

```
@@ -0,0 +1,55 @@
+      # Templates are specified as default for all clusters
+      podTemplate: default
+      serviceTemplate: default
+      dataVolumeClaimTemplate: default
+
+  templates:
+    serviceTemplates:
+      - name: default
+        generateName: "{{ .Chart.Name }}-keeper"
+        spec:
+          ports:
+          - name: zk
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)